ansible/roles/providers/tasks/aws.yml

```diff
@@ -19,11 +19,27 @@
   when:
     - ansible_os_family == "Debian"
 
+- name: install cython for flatcar
+  pip:
+    name: "cython<3.0.0"
+    executable: pip3
+    extra_args: "wheel"
+  when:
+    - ansible_distribution == "Flatcar"
+
+- name: install pyyaml
+  pip:
+    name: "pyyaml==5.4.1"
+    executable: pip3
+    extra_args: "--no-build-isolation"
```

Comment:
can you please brief documentation why extra_args `wheel` and `--no-build-isolation` for future reference.
the doc for `--no-build-isolation` is `Disable isolation when building a modern source`. but I am not sure what it means.

Reply:
I'm not sure why, I copied this answer from stack overflow and tested it. 

I added a comment linking to the answer.

Reply:
Thank you.